features/fetch-crl/config.pan

```diff
@@ -11,31 +15,53 @@ variable SITE_DEF_CERTDIR      ?= SITE_DEF_GRIDSEC_ROOT+"/certificates";
 
 # Include RPMs
 variable RPMS_CONFIG_SUFFIX ?= '';
-include { 'features/fetch-crl/rpms' + RPMS_CONFIG_SUFFIX };
+include 'features/fetch-crl/rpms' + RPMS_CONFIG_SUFFIX;
 
 # Define fetch-crl version if not defined when adding RPMs
 variable FETCH_CRL_VERSION ?= '3.0';
 
-# ----------------------------------------------------------------------------
+# ---------------------------------------------------------------------------- 
 # fetch-crl configuration
-# ----------------------------------------------------------------------------
-include { 'components/sysconfig/config' };
-"/software/components/sysconfig/files/fetch-crl/CRLDIR" = SITE_DEF_CERTDIR;
-"/software/components/sysconfig/files/fetch-crl/FORCE_OVERWRITE" = if ( FETCH_CRL_FORCE_OVERWRITE ) {
-                                                                     'yes';
-                                                                   } else {
-                                                                     'no';
-                                                                   };
-"/software/components/sysconfig/files/fetch-crl/QUIET" = if ( FETCH_CRL_QUIET ) {
-                                                           'yes';
-                                                         } else {
-                                                           'no';
-                                                         };
-
-# ----------------------------------------------------------------------------
+# ---------------------------------------------------------------------------- 
+include 'components/metaconfig/config';
+include 'features/fetch-crl/schema';
+# property daemons is not supported in older version of metaconfig but unfortunately
+# metaconfig doesn't publish its version until 15.12. Until the version is defined
+# use QUATTOR_RELEASE to determine the version.
+prefix '/software/components/metaconfig/services/{/etc/sysconfig/fetch-crl}';
+'daemons' = {
+  if ( QUATTOR_RELEASE >= '15' ) {
+    nlist('fetch-crl-boot', 'restart');
+  } else {
+    null;
+  };
+};
+'backup' = '.old';
+'module' = 'tiny';
+'contents' = {
+  SELF["CRLDIR"] = SITE_DEF_CERTDIR;
+  if ( FETCH_CRL_BOOT_IGNORE_RETRIEVAL_ERRORS ) {
+    SELF["FETCHCRL_BOOT_OPTIONS"] = '"--define rcmode=noretrievalerrors"';
+  };
+  SELF["FORCE_OVERWRITE"] = if ( FETCH_CRL_FORCE_OVERWRITE ) {
+                              'yes';
```

Comment:
No when there is a `if` the usage (mine at least!!) is to ident based on the `if` position. Much easer to read IMO...